src/Essentials/src/Flashlight/Flashlight.shared.cs

```diff
@@ -8,6 +8,12 @@ namespace Microsoft.Maui.Devices
 	/// </summary>
 	public interface IFlashlight
 	{
+		/// <summary>
+		/// Checks if the flashlight is available and can be turned on or off.
+		/// </summary>
+		/// <returns><see langword="true"/> when the flashlight is available, or <see langword="false"/> when not</returns>
+		Task<bool> CheckIsSupported();
```

Comment:
Should this have the `Async` suffix to indicate that it needs to be awaited?

Reply:
What is your policy regarding to `async` methods?

Reply:
I think we typically have Async suffix - especially since essentials has a mix.

Reply:
Ok I'll add the suffix.